### ISSUE TITLE
Is this supposed to still work with zbackup-1.4.4??

### ISSUE THREAD

**Issue by GinoM** [davidbartonau/zbackup-tar#9]
Getting the following error.. also the backup having a tar name is weird seeming to me
```
Traceback (most recent call last):
  File "/usr/local/bin/zbackup-tar", line 899, in <module>
    main()
  File "/usr/local/bin/zbackup-tar", line 863, in main
    retCode = handleBackupCreation(cParser.parse_args())
  File "/usr/local/bin/zbackup-tar", line 626, in handleBackupCreation
    createZBackup(nBackup, pArgs.aFlags)                                
  File "/usr/local/bin/zbackup-tar", line 403, in createZBackup
    nameInTar = shouldIncludeTarInfoAndRecord (True, fullFile)
  File "/usr/local/bin/zbackup-tar", line 572, in shouldIncludeTarInfoAndRecord
    printBackupFileInfo(backupFile)
  File "/usr/local/bin/zbackup-tar", line 699, in printBackupFileInfo
    print(backupFile.pathInTar + "\tSTORED")
UnicodeEncodeError: 'utf-8' codec can't encode character '\udcab' in position 33: surrogates not allowed
```

**Comment by davidbartonau** [comment]
What encoding is your filesystem?

I've seen a similar provlem in /etc/ssl/certs (or something similar).  When I "ls -l" the directory it give me a lot of ????? in the filenames.  It turns out my locale was incorrectly set and the filenames were returning illegal characters.

**Comment by davidbartonau** [comment]
The zbackup file IS a tar.  It just isn't complete.  In earlier versions where there were bugs, you could manually run zbackup and tar over the last N tarfiles from oldest to newest and get a complete restore.

So tar1.tar has A, and D, tar2 has B and E, and tar3 has A(modified), C, F.  If you just tar restore them in order, you'll get a proper restore.  Having the file as a real tarfile is a feature not a defect.

**Comment by davidbartonau** [comment]
BTW, there seems to be an unrelated problem with later versions of zbackup, (see issue #11 ).

It should be an easy fix.

**Comment by davidbartonau** [comment]
I have checked and #11 seems to be fixed with zbackup 1.4.4
